packages/common/src/store/purchase-content/utils.ts

```diff
@@ -61,12 +64,35 @@ export const getPurchaseSummaryValues = ({
     // Note: Rounding amount due *up* to nearest cent for cases where the balance
     // is between cents so that we aren't advertising *lower* than what the user
     // will have to pay.
-    const diff = amountDueBN.sub(balanceBN)
-    amountDue = ceilingBNUSDCToNearestCent(diff as BNUSDC)
+    const diff = ceilingBNUSDCToNearestCent(
+      amountDueBN.sub(balanceBN) as BNUSDC
+    )
       .div(BN_USDC_CENT_WEI)
       .toNumber()
-    existingBalance = balanceBN.div(BN_USDC_CENT_WEI).toNumber()
+    // Don't allow use of existing balance if final amount due is less than the minumum purchase amount
+    if (diff > 0 && diff >= minPurchaseAmountCents) {
+      amountDue = diff
+      existingBalance = balanceBN.div(BN_USDC_CENT_WEI).toNumber()
+    }
```

Comment:
minor nit: would be cool if we could use `getBalancedNeeded` here instead of rewriting the same logic, but nbd.

Reply:
I agree, but didn't want to mess with it too much since they return slightly different shapes of objects.